src/strategyHelpers/blueGreen/smiBlueGreenHelper.ts

```diff
@@ -178,13 +178,13 @@ export async function cleanupSMI(
    const deleteList: K8sDeleteObject[] = []
 
    serviceEntityList.forEach((serviceObject) => {
-      deleteList.push({
-         name: getBlueGreenResourceName(
-            serviceObject.metadata.name,
-            TRAFFIC_SPLIT_OBJECT_NAME_SUFFIX
-         ),
-         kind: TRAFFIC_SPLIT_OBJECT
-      })
+      // deleteList.push({
```

Comment:
better to remove than just uncomment here. people won't remember the context and it will just be confusing